src/renderer/material/material.cpp

```
@@ -1,50 +1,145 @@
-mr::graphics::Material::Material(const VulkanState &state,
-                       const RenderContext &render_context,
-                       mr::graphics::ShaderHandle shader,
-                       std::span<std::byte> ubo_data,
-                       std::span<std::optional<mr::TextureHandle>> textures,
-                       mr::UniformBuffer &cam_ubo) noexcept
-    : _ubo(state, ubo_data)
+// ----------------------------------------------------------------------------
+// Material
+// ----------------------------------------------------------------------------
+
+static size_t get_aligned_16_byte(size_t size) {
```

Cppcheck (2.19.0)

[style] The function 'get_aligned_16_byte' is never used.

(unusedFunction)